gallery/src/PageHeader/PageHeader.scss

```diff
@@ -27,21 +27,15 @@
     height: $logo-image-size;
   }
 
-  &__info {
-    color: var(--nx-swatch-grey-30);
-  }
-
   &__name {
+    color: var(--nx-swatch-grey-30);
     font-size: 18px;
     font-weight: 600;
     margin-right: var(--nx-spacing-4x);
     min-width: max-content;
   }
 
   &__version {
```

Comment:
Ah this makes me notice that the old version number is now gone.  What's expected to happen when running locally? As I mentioned in another thread currently we get the dropdown but it doesn't work.

Reply:
> Ah this makes me notice that the old version number is now gone. What's expected to happen when running locally? As I mentioned in another thread currently we get the dropdown but it doesn't work.

When running locally it will not show the dropdown: 18aab428bcb0badf4d66b08b695127dd78b9a43e